pages/[endpoint].tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import { useQuery } from '@apollo/client';
+import { Checkbox, Pivot, PivotItem } from '@fluentui/react';
+
+import { BlockList, TransactionList } from 'components/List';
+import OffsetSwitch from 'components/OffsetSwitch';
+
+import {
+  getEndpointFromQuery,
+  GRAPHQL_ENDPOINTS,
+  nullEndpoint,
+} from 'lib/graphQLEndPoint';
+import {
+  CommonPageProps,
+  getCommonStaticPaths as getStaticPaths,
+  getCommonStaticProps as getStaticProps,
+} from 'lib/staticGeneration';
+import { listTxColumns, mainMineColumns } from 'lib/listColumns';
+import useOffset, { limit } from 'lib/useOffset';
+import useSearchParams from 'lib/useSearchParams';
+
+import {
+  Block,
+  BlockListDocument,
+  BlockListQuery,
+  Transaction,
+  TransactionListDocument,
+  TransactionListQuery,
+} from 'src/gql/graphql';
+const POLL_INTERVAL = 2000;
+const ROUND_DIGITS = 4;
+
+export default function Summary({ staticEndpoint }: CommonPageProps) {
+  const [blocks, setBlocks] = useState<Block[] | null>(null);
+  const [blockList, setBlockList] = useState<JSX.Element>(<></>);
```

Comment:
I heard about treating `JSX.Element`s as state causes unexpected behavior. is it okay?

Reply:
I think it would be better to make new components in this case.
(e.g., BlockList , TransactionList)

Reply:
It did cause quite a lot of trouble, so I removed as much hook calls as I could in the newest commit including the new one. PTAL 🙏